README.md

```diff
@@ -14,13 +14,15 @@ yarn add seek-datadog-custom-metrics
 
 ## Table of contents
```

Comment:
Should we remove the ToC given GitHub generates one for us now?

<img width="341" alt="Screenshot 2023-03-18 at 7 18 20 pm" src="https://user-images.githubusercontent.com/25572311/226094092-76ed12a3-e4ad-4e2d-8a0f-1276aecdc760.png">